Libraries/LibWeb/CSS/StyleValues/ShorthandStyleValue.cpp

```diff
@@ -65,6 +65,8 @@ String ShorthandStyleValue::to_string(SerializationMode mode) const
         auto color = longhand(PropertyID::BackgroundColor);
         auto image = longhand(PropertyID::BackgroundImage);
         auto position = longhand(PropertyID::BackgroundPosition);
+        auto position_x = position->as_shorthand().longhand(PropertyID::BackgroundPositionX);
+        auto position_y = position->as_shorthand().longhand(PropertyID::BackgroundPositionY);
```

Comment:
The problem here is that `background-position` is now a shorthand of lists rather than a list of shorthands, so it needs to be unpacked before it can be indexed as the rest of the code expects.

Reply:
Gotcha :+1: